platforms/chibios/drivers/i2c_master.c

```diff
@@ -31,6 +31,9 @@
 
 static uint8_t i2c_address;
 
+#if defined(WB32F3G71xx)
+static const I2CConfig i2cconfig = WB32_I2CCONFIG;
```

Comment:
So this is being hard coded to only support this config?  

if so, does this chip support other configurations?  If so, then this should be configurable.

Reply:
This is the default configuration, the user can modify the configuration by redefining WB32_I2CCONFIG

Reply:
Ideally, would want to have the config set up like the rest of it, so that other people using the same or similar chips don't have to add the full config, just add defines based on their usage.